tests/e2e/CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing
```

Comment:
Do we still need a contributing guide?

Reply:
yes, it explains how to write tests

Reply:
That should be in our [main contributing guide](https://github.com/algolia/instantsearch.js/blob/master/CONTRIBUTING.md#tests) then.